client/utils/types.ts

```diff
@@ -9,6 +7,12 @@ export const RatingCriteria: { [key: string]: Array<string> } = {
   washrooms: ['cleanliness'],
 }
 
+export const ResourceDisplayStrings: { [key: string]: string } = {
+  housing: 'Housing',
+  'study-spots': 'Study Spots',
+  washrooms: 'Washrooms',
+}
+
```

Comment:
We can also opt for one centralized data structure which is somewhat cleaner.

```
interface ResourceInfo {
  title: string
  criteria: Array<string>
}

interface ResourceLookup {
  [name: string]: ResourceInfo
}

const resourceLookup: ResourceLookup = {
  housing: {
    title: 'Housing',
    criteria: [...],
  }
}
```

But I guess the benefit with constant objects is that you can just hard-code the data right here instead of needing to create instances of the interface. What do you think @xujustinj?

Reply:
The interface you're suggesting here is like the API types module; a meta definition-of-a-definition. I like it. We are still exporting a const instance of the interface, and it's pretty much inline hard-coded (with type-checks that prevent you from writing incorrect inline definitions), so I don't see the benefit you mention.

If we want to go super-tryhard (mainly for our own learning's sake), we'd probably eventually hook the whole thing up with i18n. Either implementation will probably need a refactor in the future to adapt to that.